features/flows/kafka-persistence/src/main/java/org/opennms/netmgt/flows/persistence/model/NodeInfoOrBuilder.java

```diff
@@ -37,44 +37,55 @@
 
   /**
    * <code>string foreign_source = 1;</code>
+   * @return The foreignSource.
    */
   java.lang.String getForeignSource();
   /**
    * <code>string foreign_source = 1;</code>
+   * @return The bytes for foreignSource.
    */
   com.google.protobuf.ByteString
       getForeignSourceBytes();
 
   /**
    * <code>string foregin_id = 2;</code>
+   * @return The foreginId.
```

Comment:
typo, should be foreignId

Reply:
The typo was not introduced by me - I've just regenerated the file from protobuf. I fixed it now but I'm uncertain if this wil break eternal users or is fine as the field ID is still the same?